### PR TITLE
[terra-time-input] Remove Terra.it helpers from wdio

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Remove Terra.it helpers from wdio as they are deprecated.

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-time-input/tests/wdio/time-input-twelve-hour-spec.js [hunk 1]
```diff
@@ -1,177 +1,123 @@
...
 
-    Terra.it.validatesElement();
+    browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+    browser.keys('12');
+    browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+    browser.keys(['ArrowDown']);
+    Terra.validates.element('switch meridiem - down arrow');
   });
 
-  describe('Twelve Hour - Sets time to 12 when hour is 0 and onBlur is called', () => {
```
Comment (mjhenkes) [inline packages/terra-time-input/tests/wdio/time-input-twelve-hour-spec.js:134]: was this a duplicate test?

Comment (saket2403) [reply packages/terra-time-input/tests/wdio/time-input-twelve-hour-spec.js:134]: Yes it was written 3 times

**Review by mjhenkes — APPROVED** [review]

**Review by pranav300 — APPROVED** [review]

File: packages/terra-time-input/tests/wdio/time-input-spec.js [hunk 2]
```diff
@@ -1,363 +1,285 @@
...
+    Terra.validates.element('seconds no time');
   });
 
-  describe('Second with time provided', () => {
-    before(() => browser.url('/#/raw/tests/terra-time-input/time-input/time-input/second-time'));
-
-    Terra.it.validatesElement();
+  it('displays Second with time provided', () => {
+    browser.url('/raw/tests/terra-time-input/time-input/time-input/second-time');
+    Terra.validates.element('second without time');
```
Comment (benbcai) [inline packages/terra-time-input/tests/wdio/time-input-spec.js:19]: Should this be "second with time"?

File: packages/terra-time-input/tests/wdio/time-input-spec.js [hunk 3]
```diff
@@ -1,363 +1,285 @@
 Terra.describeViewports('Time Input', ['medium'], () => {
-  describe('Default with no time provided', () => {
-    before(() => browser.url('/#/raw/tests/terra-time-input/time-input/time-input/default'));
-
-    Terra.it.validatesElement();
+  it('displays Default with no time provided', () => {
```
Comment (benbcai) [inline packages/terra-time-input/tests/wdio/time-input-spec.js:2]: Should lower case the second word in all the other `it` blocks.

```suggestion
  it('displays default with no time provided', () => {
```

**Review by benbcai — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**